Makefile

```diff
@@ -83,7 +74,7 @@ unittest:
 test: generate-mocks unittest
 
 .PHONY: integration-test
-integration-test: docker-build-flow
+integration-test: docker-build-flow 
```

Comment:
did anything change here except a blank space at the line end (?)

Reply:
No, it must be a left over from debugging :)